tools/hangar/__snapshots__/test_corpus/sdk_tests/fs/yaml.main.w_test_sim.md

```diff
@@ -1,5 +1,11 @@
 # [yaml.main.w](../../../../../../examples/tests/sdk_tests/fs/yaml.main.w) | test | sim
 
+## stderr.log
+```log
+(node:92057) Warning: Keys with collection values will be stringified due to JS Object restrictions: "{ content }". Set mapAsMap: true to use object keys.
+(Use `node --trace-warnings ...` to show where the warning was created)
+```
+
```

Comment:
ℹ️  https://github.com/winglang/wing/issues/4540

Reply:
Went ahead and deleted the test since it wasn't working anyways (the readYaml did not throw, so the catch assertion wasn't running)